code-studio/src/js/components/playzone.jsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * @overview React for the End-of-Stage Experience
+ */
+/* global React */
+var color = require('@cdo/apps/color');
+
+export class PlayZone extends React.Component {
```

Comment:
Our preference thus far as been to continue to use React.createClass instead of `extend React.Component`, though I'm not sure we have any great reason for it.